client/Components/Header/header.scss

```diff
@@ -1,12 +1,62 @@
-header {
+@import url('https://fonts.googleapis.com/css?family=Baloo+Bhai&display=swap');
+
+$darkBlue: #0c093c;
+$intensePink: #99235c;
+
+@mixin inline-flex {
     display: flex;
-    justify-content: space-around;
     align-items: center;
+}
+
+header {
+    @include inline-flex;
+    justify-content: space-between;
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    padding: 0 2vw;
     height: 9vh;
-    border-bottom: solid 2px;
-    border-image: radial-gradient(rgb(65,105,225), rgb(250,224,66)) 1;
+    border-bottom: solid 3px;
+    border-image: radial-gradient($darkBlue, $intensePink) 1;
+    font-family: 'Baloo Bhai', cursive;
```

Comment:
how about $dodgerBlue and $tomato? i'm sure @MariaSLapshina and prof will be thrilled...

Reply:
she can change it later. but i wont approve.

Reply:
i will ask Charm to approve then..